crates/ruff_linter/src/rules/refurb/rules/readlines_in_for.rs

```diff
@@ -84,16 +85,24 @@ fn readlines_in_iter(checker: &Checker, iter_expr: &Expr) {
             return;
         }
     }
+    let edit = if let Some(parenthesized_range) = parenthesized_range(
+        expr_attr.value.as_ref().into(),
+        expr_attr.into(),
+        checker.comment_ranges(),
+        checker.source(),
+    ) {
+        let unparenthesized_item_range = parenthesized_range.add_start(1.into()).sub_end(1.into());
+        let item_str = checker.locator().slice(unparenthesized_item_range);
+        Edit::range_replacement(item_str.to_string(), expr_call.range())
```

Comment:
Is there a better way to unpack the parenthesized expression here? I looked but didn't find any.

When the expression has multiple parenthesis like in:
```python
with open("furb129.py") as f:
    for line in (((f))).readlines():
        pass
``` 

the fix will generate this `for line in ((f)):`, would that be the correct behavior or should it be just `f`?

Reply:
Is it necessary to remove any parentheses? Removing them when fixing `for line in(f).readlines()` would produce a syntax error, similar to #17683.

Reply:
> Is it necessary to remove any parentheses?

Not really, I guess.

Reply:
> But `for line in f.readlines()` is not incorrect syntax. Syntax error itself is `for line in(f)` block: `in` is not a function and whitespace should not be removed

`for line in(f)`  is not a syntax error, `in` is lexed as a keyword and not as an identifier, so `in(f)` is not a function call.